libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/component/audioguidance/AudioGuidanceDevice.kt

```diff
@@ -0,0 +1,11 @@
+package com.mapbox.navigation.dropin.component.audioguidance
+
+import android.content.Context
+import com.mapbox.navigation.base.internal.extensions.inferDeviceLanguage
+
+/**
+ * This object is used to test context interaction, without requiring Robolectric
+ */
+internal object AudioGuidanceDevice {
```

Comment:
Can this be moved to `com.mapbox.navigation.base.internal.extensions.ContextEx`

Reply:
ContextEx already contains the function inferDeviceLanguage()

Reply:
https://github.com/mapbox/mapbox-navigation-android/blob/f4df89edb951dc24ede4087efad690723e6bd7c6/libnavigation-base/src/main/java/com/mapbox/navigation/base/internal/extensions/ContextEx.kt#L3

yeah this is actually using that extension function. But how do you mockk it? Is there an example somewhere because I couldn't find it. Seems like you have to do it through reflection. if ContextEx was an object, it would be easy to mock 

https://medium.com/@ByteSizedBit/mockk-testing-extension-functions-8b207bcad21d
```
@Test
fun `top level extension function test`() {
    mockkStatic("com.bytesizebits.extensions.MyClassKt")
    every { MyClass(1).topLevelExtension() } returns 3
    assertEquals(3, MyClass(1).topLevelExtension())
    verify { MyClass(1).topLevelExtension() }
}
```

Reply:
Look at LocaleExTest.kt

Reply:
Looks like we're reflection mockking in other places so will copy that
```
    @Before
    fun setUp() {
        mockkStatic("com.mapbox.navigation.base.internal.extensions.ContextEx")
    }
```

Reply:
You can also setup mocks this way:
```kotlin
mockkStatic(Context::inferDeviceLanguage)
```